docs/user/commandline_tools.rst

```diff
@@ -216,11 +216,16 @@ includes some examples. SFT and Converter specifications should use the path pre
 
 For example, here's a simple CSV file to ingest named ``example.csv``::
 
-    ID,Name,Age,LastSeen,Friends,Lat,Lon
+    FID,Name,Age,LastSeen,Friends,Lat,Lon
     23623,Harry,20,2015-05-06,"Will, Mark, Suzan",-100.236523,23
     26236,Hermione,25,2015-06-07,"Edward, Bill, Harry",40.232,-53.2356
     3233,Severus,30,2015-10-23,"Tom, Riddle, Voldemort",3,-62.23
 
+.. note::
+
+    ID is a reserved word, for a full list of reserved words see
+    `Geomesa Data Store Reserved Words <http://www.geomesa.org/documentation/user/datastores.html#reserved-words>`__.
```

Comment:
This hardcodes the URL for the built docs, which will break a lot of things (versioned forms of old docs, non-HTML doc builds). Better to use ``:ref:`` (see http://www.sphinx-doc.org/en/stable/markup/inline.html#role-ref). E.g. put

    :ref:`reserved_words`

here, and

    .. _reserved_words:

above the Reserved Words section of ``datastores.rst``.